cmd/auto-pause/auto-pause.go

```diff
@@ -48,21 +48,38 @@ func main() {
 	// TODO: #10595 make this configurable
 	const interval = time.Minute * 1
 
+	// Check if interval is greater than 0 so NewTicker does not panic.
+	if interval <= 0 {
+		exit.Message(reason.Usage, "Auto-pause interval must be greater than 0,"+
+			" not current value of {{.interval}}", out.V{"interval": interval.String()})
+	}
+	tickerChannel := time.NewTicker(interval)
```

Comment:
defer stop the ticker.

Reply:
Putting a defer stop the ticker give a linter error: cmd/auto-pause/auto-pause.go:93:2: exitAfterDefer: log.Fatal will exit, and `defer tickerChannel.Stop()` will not run (gocritic)
        log.Fatal(http.ListenAndServe("0.0.0.0:8080", nil))
        ^
make: *** [lint] Error 1

Reply:
Instead, I'll put it at the very end.